pubspec.yaml

```diff
@@ -125,6 +125,7 @@ dependencies:
   audioplayers: ^5.2.1
   logger: ^2.2.0
   visibility_detector: ^0.4.0+2
+  local_auth: ^2.2.0
```

Comment:
Same as any sensitive action that requires user permission, please check if Apple will throw an error/warning because we have authentication code but may not include the user requests in plist / AndroidManifest. If so we MUST move this to its own module

Reply:
Apple won't throw any error or warning if plist doesn't include permission, so no issue keep this action in main package